sections/faqs/reverse-selectors.js

```diff
@@ -70,7 +69,10 @@ const ReverseSelectors = () => md`
     ${A} {
     }
   `;
-  \`\`\`
+  ```
+
+  The error thrown - `Cannot call a class as a function` - occurs because the 
```

Comment:
Small punctuation nit; Can we use an em dash here? "—"

Reply:
@oliverturner this nit is still outstanding; looks fine to me otherwise :+1:

Reply:
Seems not: commonmark doesn't support the `---` syntax for emdash and my editor treats -  and — as the same character :(

Reply:
wut; ok, that's weird